hack/make-rules/release.sh

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+###
+#Copyright 2021 The KubeEdge Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+###
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBEEDGE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../.." && pwd -P)"
+
+source "${KUBEEDGE_ROOT}/hack/lib/golang.sh"
+
+function release() {
+  local -a targets=()
+  local VERSION=""
+  local ARCH="amd64"
+
+  for arg in "$@"; do
+    if [[ "${arg}" == GOARM7 ]]; then
+      ARCH="arm"
+    elif [[ "${arg}" == GOARM8 ]]; then
+      ARCH="arm64"
+    else
+      targets+=("${arg}")
+    fi
+  done
+
+  if [[ ${#targets[@]} -eq 0 ]]; then
+    targets+=("kubeedge")
+    targets+=("edgesite")
+    targets+=("keadm")
+  fi
+
+  kubeedge::version::get_version_info
+  VERSION=${GIT_VERSION} #got v1.9.0-54+14e4f891a66a05
+
+  for bin in ${targets[@]}; do
+    if [ "${bin}" == "edgesite" ]; then
+      hack/make-rules/build.sh edgesite-server edgesite-agent
+      echo "build edgesite release:"$VERSION "ARCH:"$ARCH
+      build_edgesite_release $VERSION $ARCH
```

Comment:
Can init the VERSION and print the log in `build_edgesite_release`, same in below

Reply:
done